content/blog/2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Moving Local Experiments to the Cloud with Terraform Provider Iterative
+date: 2022-01-15
+description: >
+  In this tutorial you'll learn how you can move a locally run machine learning
+  experiment to a remote machine on AWS with the help of Iterative's Terraform
+  Provider.
+descriptionLong: >
+  In this tutorial you'll learn how you can move a locally run machine learning
+  experiment to a remote machine on AWS with the help of Iterative's Terraform
+  Provider.
```

Comment:
Minor: Should short desc. be shorter and/or long one be longer?

From looking at https://dvc-org-tpi-tutorial-hgp3t6hg6.herokuapp.com/blog I see the short one fits well in the big post preview but will probably be truncated once it's a smaller thumbnail so I'd consider the former option mainly.

Reply:
how about now @jorgeorpinel?

Reply:
Like it!